### ISSUE TITLE
Add color palette to styleguide app

### ISSUE THREAD

**Issue by trevordmiller** [eggheadio/egghead-ui#91]
It would be nice to have the color palette rendered in the styleguide portion of egghead-ui so that if you go to http://styleguide.egghead.io you see it, with the names available to use.

Mocks:
https://egghead.invisionapp.com/d/main#/console/10384631/225150686/comments

![image](https://cloud.githubusercontent.com/assets/5497885/24313744/e21992de-10a3-11e7-9e9e-8d925b94c21b.png)

**Comment by trevordmiller** [comment]
@vojtaholik @joelhooks @tayiorbeii @DrShpongle I believe this story is ready to be implemented; is there anything we should discuss first?

**Comment by trevordmiller** [comment]
Once this tachyons story is done, we could programmatically map over the values to display them:
https://github.com/eggheadio/tachyons-egghead/issues/28
